modules/mapreduce/src/main/java/org/apache/fluo/mapreduce/FluoOutputFormat.java

```diff
@@ -44,7 +44,6 @@
 
   @Override
   public void checkOutputSpecs(JobContext arg0) throws IOException, InterruptedException {
-    // TODO Auto-generated method stub
```

Comment:
Does this change trigger any spotbugs/checkstyle failures? If the intent for the body is to be empty, a comment explaining that intent can be helpful in future, even if spotbugs/checkstyle don't flag it.

Reply:
Thanks for your comment.

Though I read the contributor's guide, I'll be honest I assumed it would not and that travis during the run of mvn verify would catch it if there were any checkstyle/findbugs issues. With that said I accidentally did some testing and changes on this branch yesterday and have built/verified successfully further validating that no issues should arise as a result of this. 

The Eclipse templates have a few auto inserts ( one appears to be rem'd ) for exceptions. I can only presume this was the case for methods. Given the name "arg0" of the comment, my assumption was that this class was stubbed out from OutputFormat and therefore there was no likely reason it existed. 

There are others throughout the code base. I was planning on making those stubbed removals as I potentially made other (unrelated) changes to those file(s), so I will create an issue for those change sets and piggy back the removal of stubs; however, I'd be just as fine creating an issue removing all stubs and isolating those changes. It could be that I totally trash my changes. My efforts are more for learning the code base than making adjustments.

My guess is that you and @keith-turner will tell me what fluo typically prefers when an issue/pr arise for those changes -- but the change here was simply a singular instance that I perceived to have little friction.

Reply:
If no build errors occur, I'm fine with it.